spec/core/data_structures.md

```diff
@@ -455,8 +466,8 @@ func SumTruncated(bz []byte) []byte {
 
 | Name          | Type   | Description                   | Field Number |
 |---------------|--------|-------------------------------|--------------|
-| message_delay | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how long a proposal message may take to reach all validators on a newtork and still be considered valid. | 1            |
-| precision     | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how skewed a proposer's clock may be from any validator on the network while still producing valid proposals. | 2            |
+| precision     | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how skewed a proposer's clock may be from any validator on the network while still producing valid proposals. | 1            |
+| message_delay | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how long a proposal message may take to reach all validators on a newtork and still be considered valid. | 2            |
 
 ### TimeoutParams
```

Comment:
The description for the timeouts was not introduced by this PR, but I have to say that I don't like it very much.

The parameters without `_delta` suffix are the initial values for the timeouts, increased by the parameter with `_delta` suffix at every new round of consensus.

Reply:
@cmwaters What are your thoughts on this one